scripts/operations/node_management/ncn-ims-image-upload.sh

```diff
@@ -122,4 +136,16 @@ cray ims images update "$IMS_IMAGE_ID" \
         --link-etag "${MANIFEST_ETAG}" \
         --link-path "s3://boot-images/$IMS_IMAGE_ID/manifest.json" > /dev/null
 
+podman run --rm --name ncn-cpc \
+    --user root \
+    -e PRODUCT=csm \
+    -e PRODUCT_VERSION=$CSM_RELEASE \
+    -e YAML_CONTENT_STRING="{images: {$IMS_IMAGE_NAME: {id: $IMS_IMAGE_ID}}}" \
+    -e KUBECONFIG=/.kube/admin.conf \
+    -e VALIDATE_SCHEMA="true" \
+    -e SET_ACTIVE_VERSION="true" \
+    -v /etc/kubernetes:/.kube:ro \
+    -v ${PWD}/catalog:/catalog:ro \
```

Comment:
If you are passing the YAML content as a string rather than a file, I don't think there is any need to mount the working directory into the container.

Reply:
Thanks - left over from testing.  Removing...